app/settings/tokens/edit/controller.ts

```diff
@@ -20,8 +21,9 @@ export default class SettingsTokensEditController extends Controller {
     @reads('model.taskInstance.value')
     token?: Token;
 
-    @action
-    refresh() {
+    @task
+    @waitFor
+    async refreshToken() {
```

Comment:
I think this can be reverted back to an `action`, since there is no async behavior in this method.

Reply:
Understood, please see the related comment regarding this.